src-vue/src/plugins/store.ts

```diff
@@ -245,12 +245,16 @@ export const store = createStore<FlightCoreStore>({
                 })
                 .catch((error) => {
                     console.error(error);
+                    // TODO discuss if we should delete this
+                    // If there's no other error case as "no install selected", I think we can get rid of it.
+                    /*
                     ElNotification({
                         title: 'Error',
                         message: error,
                         type: 'error',
                         position: 'bottom-right'
                     });
+                    */
```

Comment:
While unlikely in practice, there can be other error cases. E.g.: lack of mods directory or some incorrectly formatted JSON.

It might make more sense to do an if-guard to check if the install path is actually set, before calling the backend function ^^

Reply:
Reverted in 2f6f90ee0dcafb03853b9dd29f79c5c118dcd350